lib/jasmine-reportportal-reporter.js

```diff
@@ -101,6 +103,18 @@ class ReportportalReporter {
         }
     }
 
+    setStatus(data) {
+        if (data && data.suite) {
+            this.suiteStatus.set(data.suite, data.status);
+        } else {
+            this.additionalCustomParams = Object.assign(this.additionalCustomParams, data && { customStatus: data.status });
+        }
+    }
+
+    setLaunchStatus(status) {
+        this.conf.launchStatus = status;
```

Comment:
I'm not sure that it's good idea to modify config in runtime.

Reply:
Unfortunately, the other way with the callback did not work.